jellyfish/api/_room_api.py

```diff
@@ -4,36 +4,33 @@
 
 from typing import Literal, Union
 
-from jellyfish._openapi_client import AuthenticatedClient
-from jellyfish._openapi_client.api.hls import subscribe_tracks
-from jellyfish._openapi_client.api.room import (
-    add_component,
-    add_peer,
-    create_room,
-    delete_component,
-    delete_peer,
-    delete_room,
-    get_all_rooms,
-    get_room,
-)
+from jellyfish._openapi_client.api.hls import subscribe_tracks as hls_subscribe_tracks
```

Comment:
this is kindof ugly, but at least more readable, as we know where the functions are coming from. This could be fixed with a PR to the openapi-python-generator.

Reply:
Couldn't we make imports like this:
```python3
from jellyfish._openapi_client.api import (hls, room)
```

And then, in the code below, we would use methods like this:
```python3
return self._request(room.delete_room, room_id=room_id)
```
I don't see much benefit in adding an alias for every method.

Reply:
Unfortunately, the functions aren't exported by the `room` and `hls` modules, that's why we cannot do it. AFAIK we can't fix it, so alias is the only way.